prime-router/metadata/fhir_mapping/hl7/resource/MessageHeader.yml

```diff
@@ -78,3 +78,18 @@ extension_3:
         system: "http://terminology.hl7.org/CodeSystem/v3-AcknowledgementCondition"
       vars:
         code: MSH.16
+
+extension_4:
+  condition: $msh2 NOT_NULL && $msh2 NOT_EQUALS_STRING ^~\&
```

Comment:
Why not just set this always if NOT_NULL? My concern is that the spec allows you to pick any characters here, and the ones you are checking against are just the recommended ones? `Recommended values are ^~\& (ASCII 94, 126, 92, and 38, respectively). See Section 2.5.4, "Message delimiters'.`